apps/metadata/admin.py

```diff
@@ -170,13 +173,15 @@ class ActivityLabelAdmin(admin.ModelAdmin):
 class WealthCharacteristicLabelAdmin(admin.ModelAdmin):
     fields = (
         "wealth_characteristic_label",
+        "status",
```

Comment:
we don't have a link to `WealthCharacteristicLabelAdmin` and `ActivityLabelAdmin` in the menu so far I think, do we need to add? or just access it from the url when needed ? maybe for super admin user only ?

Reply:
@girumb , yes please add them to the menu.

Reply:
Done to this branch